src/app/styles/js/HeaderCard.js

```diff
@@ -64,12 +52,6 @@ export const StyledHelper = styled.div`
   `}
 `;
 
-export const StyledProfileCard = styled.div`
-  margin-bottom: ${units(6)};
-  background-color: white;
-  box-shadow: ${boxShadow(1)};
-`;
-
 export const StyledContactInfo = styled.div`
```

Comment:
Do we still need this?

Reply:
Yes, this is still used in the `UserInfo` component. We'll get rid of it when we get there!